src/libpgagroal/management.c

```diff
@@ -1618,6 +1642,8 @@ write_header(SSL* ssl, int fd, signed char type, int slot)
    pgagroal_write_byte(&(header), type);
    pgagroal_write_int32(&(header[1]), slot);
 
+   write_header_info(header, 0 ,"pgagroal-cli ");
```

Comment:
Second argument of `write_header_info` is a `bool`, so use `true` or `false`.

Reply:
will do

Reply:
Also note there's an extra space in `"pgagroal -cli"`